docs/components/muc-banner.md

```diff
@@ -1,5 +1,5 @@
 <script setup>
-import Basic from './demo/MucBanner/Basic.vue'
+import Basic from './demo/MucBanner/BasicExample.vue'
```

Comment:
I normally don't like mixing different tasks in a single PR but I will let this change pass.

Reply:
This was necessary to satisfy the linter.
`Component name "Basic" should always be multi-word  vue/multi-word-component-names`